index.bs

```diff
@@ -4231,6 +4231,7 @@ promises returned by its <a lt="writable stream writer">writer</a>'s {{WritableS
     return new WritableStream({
       start(controller) {
         ws.onerror = () => controller.error(new Error("The WebSocket errored!"));
+        ws.onclose = () => controller.error(new Error("The server closed the connection unexpectedly!"));
```

Comment:
OK, so this definitely needs to be fixed since you can't call controller.error() twice. Not sure about the best way to do that while balancing:

- Giving as much information to the developer as possible (e.g. bad close vs good)
- Keeping the code simple enough for an example

Will noodle on it.